plugins/q/src/data/extractor-t.js

```diff
@@ -140,38 +140,39 @@ function doIt({
   ret,
   sourceKey
 }) {
-  propsArr.forEach((prop) => {
-    const pCfg = props[prop];
+  for (let i = 0; i < propsArr.length; i++) {
+    const pCfg = props[propsArr[i]];
     const arr = pCfg.fields || [pCfg];
     let coll;
     let collStr;
     if (pCfg.fields) {
       coll = [];
       collStr = [];
     }
-    arr.forEach((p) => {
+
+    for (let j = 0; j < arr.length; j++) {
       let fn;
       let str;
       let value;
       let nodes;
       let cells;
       let label;
-      if (p.type === 'primitive') {
-        value = p.value;
-        label = String(p.value);
+      if (arr[j].type === 'primitive') {
```

Comment:
would be much easier to read if you just declared `const p = arr[j]` and use `p` as before